README.hbs

```diff
@@ -1,6 +1,6 @@
 # budget blocks
```

Comment:
hbs files are used to generate markdown, we're using JSDoc specifically to generate the markdown here

Reply:
https://handlebarsjs.com